tools/pycbc_test_suite.sh

```diff
@@ -7,14 +7,18 @@ echo -e "\\n>> [`date`] Python Major Version:" $PYTHON_VERSION
 PYTHON_MINOR_VERSION=`python -c 'import sys; print(sys.version_info.minor)'`
 echo -e "\\n>> [`date`] Python Minor Version:" $PYTHON_MINOR_VERSION
 
+# This will work from anywhere within the pycbc directory
+PYCBC_BASE_DIR=`git rev-parse --show-toplevel`
```

Comment:
I think there are problems if we assume git access in certain circumstances (e.g. it will make it hard to integrate into conda's testing if we want to improve that). Can we get this without that? 

I think one can do this because the script can know what directly it exists in, no?

Reply:
I think run from any folder - the git problem raised makes sense, so I will look at that

Reply:
I think this will work now